### PR TITLE
[HOLD] Cocinizes EmbargoReleaseService.

### LINKED ISSUE #3308: Rewrite EmbargoReleaseService to use cocina items
https://github.com/sul-dlss/dor-services-app/blob/main/app/services/embargo_release_service.rb#L37

This will depend on the VersionService (#3170) being rewritten first.

### PR DESCRIPTION
**Pull request by justinlittman**
closes #3308

## Why was this change made? 🤔
Less Fedora

## How was this change tested? 🤨

⚡ ⚠ If this change has cross service impact, ***run [integration tests](https://github.com/sul-dlss/infrastructure-integration-test)*** and/or test in [stage|qa] environment, in addition to specs. ⚡

Unit

### CODE REVIEW

File: app/services/embargo_release_service.rb [hunk 1]
```diff
@@ -48,64 +75,35 @@ def self.release_item(druid)
...
-    end
+  def release_cocina_object(cocina_object)
+    # Copy access > embargo > useAndReproductionStatement, access, download, readLocation, controlledDigitalLending to access >
+    # Remove access > embargo
+    access_props = cocina_object.access.to_h.except(:access, :download, :readLocation, :controlledDigitalLending, :useAndReproductionStatement)
+    access_props.merge!(access_props[:embargo].except(:releaseDate))
+    # Remove embargo
+    access_props.delete(:embargo)
 
-    events.add_event('embargo', release_agent, 'Embargo released')
```
Comment (jcoyne) [inline app/services/embargo_release_service.rb:103]: Are you intentionally dropping this event?

**Review by jcoyne — APPROVED** [review]
Looks good, so long as we don't care about that event record.  I suspect the version writing into the event is sufficient.

File: app/services/embargo_release_service.rb (hunk 1 above)
Comment (justinlittman) [reply app/services/embargo_release_service.rb:103]: This is for the events datastream, so intentionally dropping.

**Comment by justinlittman** [comment]
Testing completed by @andrewjbtw

**Pull request merged**